src/checks/conciseness/utils

```diff
@@ -0,0 +1 @@
+/workspace/check-functions/src/utils
```

Comment:
If you are making symlinks in repositories, they should be relative (that's why I had specifically mentioned the relative locations in the script), since this symlink would not work if for any reason this absolute path isn't correct.

On another note, since the create symlink script is being run before every deploy so it may not affect, so you call